src/platform/errors/interactiveCellResultError.ts

```diff
@@ -3,7 +3,16 @@
 
 import { NotebookCell } from 'vscode';
 
-// Small helper error to use in our class
+/**
+ * Error thrown when running cells in the interactive window.
+ *
+ * Cause:
+ * Kernel.executeCell() returns an error when running a cell in the interactive window.
```

Comment:
I could go into more detail here.

Reply:
An example would probably be the most useful. The details can change, so just searching for usages is probably the best method for digging in more